readme.md

```diff
@@ -11,7 +11,7 @@
 There is no need to install this package directly.
 
 ```sh
-npm init fastify
+npm init fastify [yourapp]
```

Comment:
```suggestion
npm init fastify [your_app_name]
```

Reply:
Should fastify-cli be updated to match?

https://github.com/fastify/fastify-cli?tab=readme-ov-file#generate

```
`fastify-cli` can also help with generating some project scaffolding to kickstart the development of your next Fastify application. To use it:

1. `fastify generate <yourapp>`
2. `cd yourapp`
3. `npm install`
```